debezium-core/src/main/java/io/debezium/transforms/outbox/EventRouterConfigDefinition.java

```diff
@@ -0,0 +1,176 @@
+         *
+         * @param value        the configuration property value; may not be null
+         * @param defaultValue the default value; may be null
+         * @return the matching option, or null if no match is found and the non-null default is invalid
+         */
+        public static InvalidOperationBehavior parse(String value, String defaultValue) {
+            InvalidOperationBehavior mode = parse(value);
+            if (mode == null && defaultValue != null) mode = parse(defaultValue);
+            return mode;
+        }
+    }
+
+    static final Field FIELD_EVENT_ID = Field.create("table.field.event.id")
+            .withDisplayName("Event ID Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("id")
+            .withDescription("The field which contains the Event ID within the outbox table");
+
+    static final Field FIELD_EVENT_KEY = Field.create("table.field.event.key")
+            .withDisplayName("Event Key Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDescription("The field which contains the Event Key within the outbox table");
+
+    static final Field FIELD_EVENT_TYPE = Field.create("table.field.event.type")
+            .withDisplayName("Event Type Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("type")
+            .withDescription("The field which contains the Event Type within the outbox table");
+
+    static final Field FIELD_PAYLOAD = Field.create("table.field.payload")
+            .withDisplayName("Event Payload Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("payload")
+            .withDescription("The field which contains the Event Type within the outbox table");
+
+    static final Field FIELD_PAYLOAD_ID = Field.create("table.field.payload.id")
+            .withDisplayName("Event Payload ID Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("aggregateid")
+            .withDescription("The field which contains the Payload ID within the outbox table");
+
+    static final Field FIELD_PAYLOAD_TYPE = Field.create("table.field.payload.type")
+            .withDisplayName("Event Payload Type Field")
+            .withType(ConfigDef.Type.STRING)
+            .withWidth(ConfigDef.Width.MEDIUM)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault("aggregatetype")
```

Comment:
Is this the correct default?

Reply:
That's related to your previous question, beign the generic name for the aggregate type. Do you have another suggestion?
the name I took from your blogpost! hehe

Reply:
No, it make sense, I just was missing the semantics at first. Thanks!